codesign/lib/src/transfer_utils.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file/file.dart';
+import 'package:process/process.dart';
+
+import './utils.dart';
+
+/// Utility function class to handle upload/download of files from/to google cloud.
+class EngineArtifactTransfer {
```

Comment:
Best practice for file names is to be named after the class they're in. Utils is generally a bad name as it's very vague

`engine_artifact_transfer.dart`

Reply:
Sure, renamed to GoogleCloudStorage.dart based on https://github.com/flutter/cocoon/pull/2125#discussion_r975895105